components/textfield/stories/textfield.stories.js

```diff
@@ -16,6 +18,35 @@ export default {
 			},
 			control: "boolean",
 		},
+		displayLabel: {
+			name: "Display label",
```

Comment:
Can you clarify display and side label?  Does it describe where the label appears relative to the textfield? Thinking about developers exploring the knobs and not understanding what it might do.

Reply:
thanks for point this out, I agree it's a bit confusing. 

I refactored it so we have controls for displayLabel, with an updated description of `Display field label` and labelPosition, `top` or `side`.